### PR TITLE
Mention encryption in CC release notes

### PR DESCRIPTION
**Pull request by abstratt**
Issue: #23866

### CODE REVIEW

**Comment by abstratt** [comment]
@bot-gradle test and merge

**Comment by abstratt** [comment]
@hythloda Please do chime in, I will add your feedback in a new change set.

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -40,6 +40,46 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
...
+
+### Configuration cache improvements
+
+TODO - Java lambdas are supported, and unsupported captured values are reported.
+TODO - File collections queried at configuration time are treated as configuration inputs.
+TODO - File system repositories are fully supported including dynamic versions in Maven, Maven local, and Ivy repositories
+
+#### Configuration cache is now encryptable
+
+The configuration of a build may retain sensitive information, including credentials. This release adds support for encryption of
```
Comment (hythloda) [inline subprojects/docs/src/docs/release/notes.md:75]: I think it is nice to add full sentence on a common use case
```suggestion
The configuration of a build may retain sensitive information. This information must remain secure.
For example, login credentials are often part of the build, but we want them to be something other than directly readable in the cache.
This release adds support for encryption of configuration cache state on disk.
```

**Comment by abstratt** [comment]
@bot-gradle cancel

**Comment by abstratt** [comment]
> @hythloda Please do chime in, I will add your feedback in a new change set.

Added your suggestions, thanks!

**Comment by abstratt** [comment]
This is going in instead through a team-wide PR. Closing.

**Pull request closed without merging**